parlai/messenger/core/messenger_manager.py

```diff
@@ -541,3 +541,13 @@ def observe_message(self, receiver_id, text, quick_replies=None):
     def observe_payload(self, receiver_id, data):
         """Send a payload through the message manager"""
         return self.message_socket.send_fb_payload(receiver_id, data)
+
+    def get_attachment_upload(self, payload):
```

Comment:
unclear function name, `upload_attachment` here and maybe use `upload_fb_attachment` in the message socket if you don't want to duplicate names? (I really don't have a problem with both being `upload_attachment` though)